tests/conftest.py

```diff
@@ -1,7 +1,13 @@
 import asyncio
 import os
+import sys
 
 from twisted.internet import asyncioreactor
 
 os.environ['HATHOR_CONFIG_FILE'] = os.environ.get('HATHOR_TEST_CONFIG_FILE', 'hathor.conf.unittests')
+
+if sys.platform == 'win32':
+    asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())
```

Comment:
Why do we need this? Add a comment explaining, please.

Reply:
I'll do. It boils down to the default asyncio loop policy changing on Python-3.8 and twisted not supporting the new default loop policy, so we have to manually set the old default (which is still well supported up to the latest version of Python).

Reply:
Done.

Reply:
Interesting. I didn't know about the change in the loop policy on Python 3.8. Thanks!